src/Functions/FunctionsStringDistance.cpp

```diff
@@ -173,6 +206,9 @@ using FunctionByteHammingDistance = FunctionsStringSimilarity<FunctionStringDist
 
 using FunctionByteEditDistance = FunctionsStringSimilarity<FunctionStringDistanceImpl<ByteEditDistanceImpl>, NameEditDistance>;
 
+struct NameJaccardIndex { static constexpr auto name = "jaccardIndex"; };
```

Comment:
Please format this like `NameByteHammingDistance` and `NameEditDistance` above.

And why is this called `jaccardIndex` and not `byteJaccardIndex`?